apps/spruce/src/components/FilterBadges/FilterBadge.tsx

```diff
@@ -44,47 +40,30 @@ const FilterBadge: React.FC<FilterBadgeProps> = ({ badge, onClose }) => {
         </StyledTooltip>
       )}
     >
-      <PaddedBadge
-        key={`filter_badge_${badge.key}_${badge.value}`}
-        data-cy="filter-badge"
-      >
-        <BadgeContent>
-          {badge.key}: {trimmedBadgeName}
-        </BadgeContent>
-        <ClickableIcon data-cy="close-badge" glyph="X" onClick={onClose} />
-      </PaddedBadge>
+      <span>
+        <PaddedChip
+          data-cy="filter-badge"
+          label={
+            <ChipLabel>
+              {badge.key}: {trimmedBadgeName}
+            </ChipLabel>
+          }
+          onDismiss={onClose}
+          variant="gray"
+        />
+      </span>
     </ConditionalWrapper>
   );
 };
 
-const ClickableIcon = styled(Icon)`
-  position: absolute;
-  right: ${size.xxs};
-  :hover {
-    cursor: pointer;
-    color: ${gray.light1};
-  }
+const ChipLabel = styled.span`
+  text-transform: uppercase;
+  font-weight: 700;
```

Comment:
Can we remove this styling from the chip? Specifically I think the text-transform has always been confusing because it seems like a different search is applied than what the user has input

Reply:
I create this ticket: https://jira.mongodb.org/browse/DEVPROD-12590